drivers/wifi/nrfwifi/Kconfig.nrfwifi

```diff
@@ -69,6 +69,9 @@ config NRF70_SCAN_ONLY
 config NRF70_RADIO_TEST
 	bool "Radio test mode of the nRF70 driver"
 
+config NRF70_OFFLOADED_RAW_TX
+	bool "Offloaded raw Tx mode of the nRF70 driver"
```

Comment:
it needs a elaborate description with help as it's a new feature.

Reply:
It's kconfig option only. Does it really need a detailed feature description?